cms/check_rules_states_by_account

```diff
@@ -0,0 +1,82 @@
+#!/usr/bin/env python
+# Copyright 2012-2020 CERN
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#    http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+#
+# Authors:
+# - Fernando Garzon, <oscar.fernando.garzon.miguez@cern.ch>, 2021
+
+"""
+Probe that counts the number of stuck, replicating rules and waiting for approval rules, clasiffied by account.
+The accounts used to classify the rules are sotred in the accounts list.
+"""
+
+from rucio.common.types import InternalAccount
+from rucio.db.sqla.constants import RuleState
+import sys
```

Comment:
Please organize these. System stuff comes first (sys) then a line break, then prometheus then the rucio stuff in order.

Reply:
addressed